test/scenarios/managed-network/output/src/managed-network/azext_managed_network/generated/_help.py

```diff
@@ -45,8 +45,8 @@
 criptionC/resourceGroups/myResourceGroup/providers/Microsoft.Network/virtualNetworks/myVirtualNetwork\\"},{\\"id\\":\\"\
 /subscriptions/subscriptionC/resourceGroups/myResourceGroup/providers/Microsoft.Network/virtualNetworks/myVirtualNetwor\
 k2\\"}],\\"subnets\\":[{\\"id\\":\\"/subscriptions/subscriptionC/resourceGroups/myResourceGroup/providers/Microsoft.Net\
-work/virtualNetworks/myVirtualNetwork3/subnets/default\\"},{\\"id\\":\\"/subscriptions/subscriptionC/resourceGroups/myR\
-esourceGroup/providers/Microsoft.Network/virtualNetworks/myVirtualNetwork3/subnets/default\\"}]}" --name \
+work/virtualNetworks/myVirtualNetwork3/subnets/mySubnet\\"},{\\"id\\":\\"/subscriptions/subscriptionC/resourceGroups/my\
+ResourceGroup/providers/Microsoft.Network/virtualNetworks/myVirtualNetwork3/subnets/mySubnet2\\"}]}" --name \
```

Comment:
I wonder if the mySubnet is replaceable? should it be some like {subnets}?

Reply:
The replacable name is used in test, not in the help message. 
For instance the ""myVirtualNetwork3" here is not in replacable format.